test/unit/run.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
```

Comment:
Unfortunately, this shell script can only be executed if the current working directory equals is `test/unit`. I would suggest to replace the whole script by an composer test script. Simply add the following lines in the `composer.json` file (directly after the name attribute):
```
"scripts": {
        "test": "phpunit test/unit"
    },
```

Then the tests can be executed with the command `composer test`.